### ISSUE TITLE
Swift example crashed at `return self.senderDisplayName()`

### ISSUE THREAD

**Issue by qiaoshun8888** [jessesquires/JSQMessagesViewController#1728]
## New issue checklist

- [ ] I have read all of the [`README`](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/README.md), [documentation](http://cocoadocs.org/docsets/JSQMessagesViewController/), and [FAQ](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/Documentation/faq.md).
- [ ] I have reviewed the [contributing guidelines](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/.github/CONTRIBUTING.md). Confirmation: ____
- [ ] I have searched [existing issues](https://github.com/jessesquires/JSQMessagesViewController/issues?q=is%3Aissue+sort%3Acreated-desc) and **this is not a duplicate**.

### General information

- Library version(s):
- iOS version(s): Simulator (iPhone 6s Plus)
- Devices/Simulators affected:
- Reproducible in the demo project? (Yes/No): 
- Related issues:

## Bug report

Run swift example then try to send a message. Demo crashed on this line:
```
    override func senderDisplayName() -> String {
        return self.senderDisplayName()
    }
```
https://github.com/jessesquires/JSQMessagesViewController/blob/develop/SwiftExample/SwiftExample/ChatViewController.swift#L289

#### Expected behavior
I think what you meant to do is this:
```
    override func senderId() -> String {
        return User.Wazniak.rawValue
    }
    
    override func senderDisplayName() -> String {
        return getName(User.Wazniak)
    }
```
Since  you are also using user Wazniak above.

`return self.senderDisplayName()` causes dead loop.

**Comment by GianniCarlo** [comment]
mmmmm I remember setting it correctly to Wazniak, some commit afterwards must've changed it, I'll add a test about it to avoid future problems

**Comment by GianniCarlo** [comment]
This was introduced on https://github.com/jessesquires/JSQMessagesViewController/commit/003740e8 about showing and hiding `senderDisplayName` @MacMeDan there's a private var `displayName` that is not used, was that what you wanted to return on `senderDisplayName()` ? let me know your thoughts about this issue 👍

**Comment by mrtom** [comment]
I just came across this and I have a fix (it was easy). Diff incoming!

**Comment by jessesquires** [comment]
fixed by #1731

**Comment by qiaoshun8888** [comment]
JSQMessage is very easy to use. Thanks guys!
